joule-labs/joule-extension
# Add version warning for LND < 0.7.1

### Description

Adds a warning if you're on a vulnerable version of LND. Refactors warnings to be a little easier to add them in the future.

### Steps to Test

1. Run an old version of LND, or manually bump the version it compares for
2. Open Joule, confirm you get the red error message
3. Click more info, confirm it pops open the modal

### Screenshots

<img width="413" alt="Screen Shot 2019-09-11 at 2 02 27 PM" src="https://user-images.githubusercontent.com/649992/64726844-1b086600-d49d-11e9-90b4-80305897b24c.png">
<img width="412" alt="Screen Shot 2019-09-11 at 2 02 38 PM" src="https://user-images.githubusercontent.com/649992/64726858-2065b080-d49d-11e9-8d98-58ff39b18103.png">
